_validate/addonVersion_schema.json

```diff
@@ -226,7 +228,17 @@
 				"https://github.com/nvaccess/addon-datastore/license.MD"
 			],
 			"title": "The URL of the license",
-			"type": "string"
+			"type": ["string", "null"]
+		},
+		"changelog": {
+			"$id": "#/properties/changelog",
+			"default": "",
+			"description": "Changes between the previous and the current version",
+			"examples": [
+				"New feature"
+			],
+			"title": "Add-on changelog (en)",
+			"type": ["string", "null"]
```

Comment:
rather than null, we should just not provide the changelog to JSON if it's not present